pitop/camera/camera.py

```diff
@@ -187,6 +190,8 @@ def __process_camera_output(self):
             try:
                 self.__frame_handler.frame = self.__camera.get_frame()
                 self.__new_frame_event.set()
+                if callable(self.on_frame):
+                    self.on_frame(self.__frame_handler.frame)
```

Comment:
it would be good for on_frame to have option to have the callback in the opencv format, like the other apis. I can't think how we do that in this case though, except having separate funcs like on_cv_frame, on_pil_frame? or some option that applies to the whole camera instance eg camera.format.